### DISCUSSION TITLE
非酋

### DISCUSSION THREAD

**Discussion by Lianlindesu** [share121/inter-knot#877] [其他]
艾梅莉埃武器池歪了两次，望周知〒▽〒
![祐子](https://github.com/user-attachments/assets/940cc078-ba4c-4cfc-8017-9cac4c2385ca)

**Comment by asdfghzxc12** [comment]
惨，以及你的图片没有加载完成你就点了上传，所以图片并不能显示出来，你需要耐心等待它加载完再点上传，示例如下：

![Uploading 图片.png…]\()
//上面这种情况就是没有加载完，很明显的是它有一个前缀“Uploading”，而且圆括号是空的。

![图片]\(https://github.com/user-attachments/assets/46e1bf05-ab62-44bb-8b7c-9bc800e7eb47)
//上面这种情况就是加载完了，很明显的是它的圆括号里是有一个GitHub的链接的。
